osp/utils/zulip_api.py

```diff
@@ -5,13 +5,32 @@
 client = Client(config_file="download")
 
 
-def get_zulip_user(zulip_id):
+def get_zulip_user(zulip_id: int) -> dict:
```

Comment:
I want your suggestion @isabelcosta  in this as we are not using this type of style in code writing somewhere else in the code of this repo.

Reply:
are you suggesting not using type hints or not this kind of type hints? @codesankalp 
on mentorship-backend we use type hints and I think it's a good practice we should start using. 

I created this issue specifically because this class is very isolated and our interface with Zulip API. I wish to have type hints added to the rest of the project. What is your opinion?

Reply:
I commented because we are not using type-hints in any other code of this repo. So I thought that this will break our code writing guidelines.
I know it's a good practice and I also think it will be good to have type-hints in all code of the project.
Now I get it and will ensure to use type-hints in the future.
Thanks for your descriptive answer @isabelcosta .

Reply:
Awesome! @codesankalp glad you agree! Maybe we should include that in the project's documentation about it (docusaurs). Do we have a file documenting our coding style? if not we could create. Also we need issues to create for the project. feel to create or delegate the creation of these.